<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductForm</class>
 <widget class="QWidget" name="ProductForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBoxcategoryId">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>126</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>110</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Unit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSalePrice">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>216</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxUnit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>156</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxManufacturer">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>96</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>189</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>MRP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMrp">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>186</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCode">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>66</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>249</y>
     <width>110</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Whole Sale Price</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>69</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Code</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>219</y>
     <width>110</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Sale Price</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditWholeSalePrice">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>246</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>110</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Category</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelete">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>310</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>99</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelManufacturer">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>14</y>
     <width>90</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Manufacturer</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCodeValid">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Code may be empty or contains text or greater than 10000</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelNameValid">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name may be empty or exceeds 20 characters limit</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelMrpValid">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mrp may be empty or contains text</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSalePriceValid">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sale Price may be empty or contains text</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWholeSalePriceValid">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Whole Sale price may be empty or contains text</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditCode</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>comboBoxcategoryId</tabstop>
  <tabstop>comboBoxManufacturer</tabstop>
  <tabstop>comboBoxUnit</tabstop>
  <tabstop>lineEditMrp</tabstop>
  <tabstop>lineEditSalePrice</tabstop>
  <tabstop>lineEditWholeSalePrice</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>pushButtonDelete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
